INSTALL_linux.sh

```diff
@@ -12,36 +12,33 @@ echo '  #######              We aim to fix that.'
 
 sleep 1;
 echo ''
-echo '[*] Installing Rbenv'
+export PATH="$HOME/.rbenv/bin:$PATH"
```

Comment:
Here, it is possible that rbenv doesn't exist, wouldn't that throw an error? I've used that script on Debian 10 without any issues, but as the installation of RoR apps can be annoying, if this is something that makes sense, no problem for me.

Reply:
Adding a non-existing directory to `PATH` is safe, if pointless. I need it added because `rbenv-installer` (really, `rbenv-doctor`) has a bug where it expects that directory to be in the PATH, but does not actually add it, itself. Ideally, I'd check if the directory was already present, and only add it if not, but, shrug -- it's still OK this way.

Reply:
Good for me then, merging!